integration-tests/src/test/java/oracle/kubernetes/operator/utils/DbUtils.java

```diff
@@ -45,6 +47,84 @@ public static OracleDB createOracleDB(String dbPropsFile) throws Exception {
 
     return oracledb;
   }
+  
+  /**
+   * create Oracle DB pod and service in the k8s cluster default namespace.
+   *
+   * @throws Exception - if any error occurs when creating Oracle DB pod and service
+   */
+  public static void startOracleDB() throws Exception {
+  	/*String cmd1 = "sh " +
+        BaseTest.getResultDir()
+        + "/create-rcu-schema/start-db-service.sh -i container-registry.oracle.com/database/enterprise:12.2.0.1";*/
```

Comment:
remove commented code

Reply:
I will ...and I will sync to the latest develop branch also after passing review. Thanks!